src/sentry/sentry_apps/webhooks.py

```diff
@@ -0,0 +1,76 @@
+"""
+Webhook broadcasting utilities for Sentry applications.
+"""
+
+import logging
+from typing import Any
+
+from sentry.sentry_apps.logic import consolidate_events
+from sentry.sentry_apps.metrics import SentryAppEventType, SentryAppWebhookFailureReason
+from sentry.sentry_apps.services.app import app_service
+from sentry.sentry_apps.tasks.sentry_apps import send_resource_change_webhook
+from sentry.sentry_apps.utils.errors import SentryAppSentryError
+
+logger = logging.getLogger(__name__)
+
+
+def broadcast_webhooks_for_organization(
```

Comment:
Should this function be its own task?

Reply:
It can be, originally for seer's purposes it would be nice to wait for feedback if it errors/succeeds, but now that I think of it for webhooks it's not like we'd fail seer etc. for it so I can make it a task.